tasks/eth/base-002-fp-upgrade/.env

```diff
@@ -0,0 +1,8 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+SUPERCHAIN_CONFIG_ADDR=0x95703e0982140d16f8eba6d158fccede42f04a4c
+OWNER_SAFE=0x7bB41C3008B3f03FE483B28b8DB90e19Cf07595c
+# This is Base's safe but the Nested.just file uses council as a keyword and
+# we want to minimize changes
+COUNCIL_SAFE=0x9855054731540A48b28990B63DcF4f33d8AE46A1
+FOUNDATION_SAFE=0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A
+SAFE_NONCE="2"
```

Comment:
NITS: Maybe adding a comment for clarity about the nonce of which safes here (cf the `OWNER_SAFE` that is the `ProxyAdminOwner`.